packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankYapily/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { PureComponent } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { compose } from 'redux'
+
+import Flyout, { duration, FlyoutChild } from 'components/Flyout'
+import { selectors } from 'data'
+import { RootState } from 'data/rootReducer'
+import { AddBankStepType } from 'data/types'
+import ModalEnhancer from 'providers/ModalEnhancer'
+
+import { ModalPropsType } from '../../../types'
```

Comment:
it could be 
```
import { ModalPropsType } from 'blockchain-wallet-v4-frontend/src/modals/types'
```
it's up to use to see which approach we do like more

Reply:
👌🏻